### PR TITLE
3901 preferences

### PR DESCRIPTION
**Pull request by sergiomilici**

### CODE REVIEW

File: www/templates/menu.html [hunk 1]
```diff
@@ -19,13 +19,21 @@ <h1 class="title">{{ 'mainMenu' | translate }}</h1>
...
           {{ 'signOut' | translate }}
         </ion-item>
+
         <ion-item>&nbsp;</ion-item>
-        <ion-item menu-close ng-click="redirectHome()">
-          {{ 'home' | translate }}
-        </ion-item><ion-item menu-close ng-click="changeCompany()">
+
+        <ion-item menu-close ng-click="redirectPreferences()">
+          Preferences
```
Comment (jpdimond) [inline www/templates/menu.html]: i18n?

File: www/js/services/user_service.js [hunk 2]
```diff
@@ -33,7 +33,11 @@ app.service('UserService', function($q, $http, $httpParamSerializer, RequestPara
         throw "Seller does not have deviceID";
       }
       this.seller = seller;
-      $rootScope.$emit('sellerLogin');
+
+      $timeout(function() {
+        $rootScope.$emit('sellerLogin');
+      },1000);
```
Comment (jpdimond) [inline www/js/services/user_service.js]: So now the seller account logs out after a minute? Is that what this does?

Comment (jpdimond) [inline www/js/services/user_service.js]: Does the 'stay signed in' preference change that at all?

Comment (sergiomilici) [inline www/js/services/user_service.js]: no. This for loading the seller from localStorage. It's firing an event called 'sellerLogin', to let all interested that the seller is logged in now.

I didn't do anything on the preference "stay signed in", but if it's for keep the seller logged in or not,  we can use this.

**Pull request merged**